book_inventory.c

```diff
@@ -0,0 +1,54 @@
+#include <stdio.h>
+#include <string.h>
+
+#define LENGTH_TITLE (41)
+#define LENGTH_AUTHOR (31)
+
+char *s_gets(char *state_book, int number_of_book);
```

Comment:
This is one of the issues with using function prototypes: `s_gets` was changed below, but it still has the old parameter names here. That can make it confusing for someone trying to use the function without looking at the definition.